### PR TITLE
Add hash links 

### LINKED ISSUE #11: 12 - Hash: Add links to explain difference between HashTables & HashMaps
I know there is an explanation that someone wrote but I think it's more helpful if they can get a concrete definition of what a hashtable and hashmap are.

### PR DESCRIPTION
**Pull request by Macro51337**
fixes #11

### CODE REVIEW

File: docs/labs/12-Hash.md [hunk 1]
```diff
@@ -55,34 +55,15 @@ between two items.
 Java has a ```HashTable``` class that is
 nearly identical to the ```HashMap``` class,
 but ```HashTable``` is now obsolete and still available for legacy reasons.
+Click [here](https://www.geeksforgeeks.org/differences-between-hashmap-and-hashtable-in-java/)
```
Comment (onzfonz) [inline docs/labs/12-Hash.md:58]: we aren't using geeks for geeks links (IMO it is SEO bot-fodder)

**Review by onzfonz — CHANGES_REQUESTED** [review]
I'd also like a second opinion on lines 65-85 being wordy
I think there are some terms that I'd like folks to understand here

Namely, that hashmaps are like dictionaries
what a key and value are and how they work
and to link it back to the example

**Comment by onzfonz** [comment]
you also need to edit your file so that it links your issue with the PR.  Please edit that in your description

**Comment by Macro51337** [comment]
added better links from oracle java...decided to add back lines 65-85 because they are helpful for students. couldn't find a way to make the text less wordy

**Review by onzfonz — CHANGES_REQUESTED** [review]
You may also just start with the hashmap link at the beginning when it first gets introduced.  That has been another way to introduce the concept and links without adding and addition sentence or you could say something like

Hashmap ([more info](https://docs.oracle.com/javase/8/docs/api/java/util/HashMap.html))

**Comment by Macro51337** [comment]
made changes requested by professor

File: docs/labs/12-Hash.md [hunk 2]
```diff
@@ -52,9 +52,10 @@ implements the ```Map``` interface.
 a bond,
 or mapping,
 between two items.
-Java has a ```HashTable``` class that is
-nearly identical to the ```HashMap``` class,
-but ```HashTable``` is now obsolete and still available for legacy reasons.
+Java has a [`HashTable`](https://docs.oracle.com/javase/8/docs/api/java/util/Hashtable.html)
+class that is nearly identical to the
+[`HashMap`](https://docs.oracle.com/javase/8/docs/api/java/util/HashMap.html) class,
+but `HashTable` is now obsolete and still available for legacy reasons.
```
Comment (onzfonz) [inline docs/labs/12-Hash.md:58]:
```suggestion
but `HashTable` is now obsolete;
it is still available for legacy reasons.
```

**Review by onzfonz — CHANGES_REQUESTED** [review]
please use commit suggestion box in the pull request on github site to commit this and I will approve it

File: docs/labs/12-Hash.md (hunk 2 above)
Comment (Macro51337) [reply docs/labs/12-Hash.md]: made changes to lines 58-59

File: docs/labs/12-Hash.md [hunk 3]
```diff
@@ -52,9 +52,11 @@ implements the ```Map``` interface.
...
 or mapping,
 between two items.
-Java has a ```HashTable``` class that is
-nearly identical to the ```HashMap``` class,
-but ```HashTable``` is now obsolete and still available for legacy reasons.
+Java has a [`HashTable`](https://docs.oracle.com/javase/8/docs/api/java/util/Hashtable.html)
+class that is nearly identical to the
+[`HashMap`](https://docs.oracle.com/javase/8/docs/api/java/util/HashMap.html) class,
+but `HashTable` is now obsolete;
+and still available for legacy reasons.
```
Comment (onzfonz) [inline docs/labs/12-Hash.md:59]:
```suggestion
it is still available for legacy reasons.
```

**Review by onzfonz — CHANGES_REQUESTED** [review]
Please make sure that you commit this from <https://github.com/comp129/comp55/pull/31>
Otherwise, I'll request another change before approving

**Review by onzfonz — APPROVED** [review]

**Pull request merged**